app/src/main/res/values-ar-rSA/strings_panel.xml

```diff
@@ -25,7 +25,7 @@
     <string name="panel_domain_forwarded_toast">تَمَّت الإضافة إلى القائمة البيضاء.</string>
     <!-- When a domain is blocked -->
     <string name="panel_domain_blocked">&lt;b&gt;%s&lt;/b&gt; محظور</string>
-    <string name="panel_domain_blocked_desc">تَمّ حَظْر هذا الموقع. للسماح به في المستقبل ، استخدم الزرّ أدناه.</string>
+    <string name="panel_domain_blocked_normal_desc">تَمّ حَظْر هذا الموقع. للسماح به في المستقبل ، استخدم الزرّ أدناه.</string>
```

Comment:
Please consider not changing existing string keys in the future, as it makes all translators have to manually update this translations everywhere in the world.

Reply:
should I change it back?